src/shared/calendar/calendar-body.tsx

```diff
@@ -0,0 +1,35 @@
+import { format, isSameDay, type EachDayOfIntervalResult } from 'date-fns'
+import { cn } from '../utils'
+
+interface CalendarBodyProps {
```

Comment:
CalendarBody 라는 파일이기에, 다음과 같이 해보는것도 좋을것 같아요. 이러면 타이핑을 많이 안해도 됩니다!

```suggestion
interface Props {
```